manifests/g/Gnu/Icecat/115.3.1esr/Gnu.Icecat.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.5.0

```

Comment:
There is no such file called Gnu under manifests/g/ , please make sure the oath that you are referring...